ngraph/core/include/ngraph/validation_util.hpp

```diff
@@ -234,6 +234,7 @@ namespace ngraph
     /// \brief Try to compute the maximum value of value
     /// \return (true, max_value) if can be determined, or (false, numeric_limits<uint64_t>::max())
     /// if not.
+    NGRAPH_DEPRECATED("Use evaluate_upper_bound: it would return HostTensorPtr to the value instead of a pair")
```

Comment:
Can we also insert doxygen deprecated attribute?

Reply:
IDK if the way I did it is the right way to put it